ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_random.py

```diff
@@ -98,6 +98,35 @@ def test_paddle_poisson(dtype_and_x, frontend, test_flags, fn_tree):
     )
 
 
+
+@handle_frontend_test(
+    fn_tree="paddle.rand",
+    input_dtypes=st.sampled_from(["int32", "int64"]),
+    shape=helpers.get_shape(
+        allow_none=False, min_num_dims=1, max_num_dims=1, min_dim_size=2
+    ),
+    dtype=st.sampled_from(["float32", "float64"]),
```

Comment:
```suggestion
    dtype=helpers.get_dtypes("valid"),
```
we should input all expected dtypes and skip the unsupported ones using the decorator suggested above